release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -304,6 +304,12 @@ For more information, see xref:../authentication/managing_cloud_provider_credent
 
 In {product-title} 4.8, the `rhcos4-moderate` profile is now complete. The `ocp4-moderate` profile will be completed in a future release.
 
+[discrete]
+[id="ocp-4-8-haproxy-upgrade-2.2.13"]
+=== Ingress Controller upgraded to HAProxy 2.2.13
```

Comment:
Checking if this should be x4 `====` per the other technical change in this section, and after checking the 4.7 release notes for reference. 😄  Otherwise, LGTM!

Reply:
You're right! Thank you!